modules/scanning-images.adoc

```diff
@@ -20,6 +20,43 @@ Scanner identifies the vulnerabilities in the:
 * programming language specific dependencies
 * programming runtimes and frameworks
 
+[discrete]
+== Understanding and addressing the commonly occurring Scanner warning messages
+
+When scanning images with {rh-rhacs-first}, you might see the `CVE DATA MAY BE INACCURATE` warning message.
+Scanner displays this message when it cannot retrieve complete information about the operating system or other packages in the image.
+
+This occurs because the image is based on an operating system or package manager that Scanner does not support, or because the operating system or package manager version has reached end-of-life and no longer receives security updates.
+
+It is important to address this warning to ensure that vulnerability reports are as accurate as possible.
+If you ignore the warning, vulnerability reports become inaccurate and your cluster remains vulnerable to security threats.
+
+.Warning messages
+[%autowidth]
+|===
+| *Message* | *Description*
+
+|`Unable to retrieve the OS CVE data, only Language CVE data is available.`
+|Indicates that Scanner does not officially support the base operating system of the image; therefore, it cannot retrieve CVE data for the operating system-level packages.
+
+|`Stale OS CVE data.`
+|Indicates that the base operating system of the image has reached end-of-life, which means the vulnerability data is outdated. For example, Debian 8 and 9.
+
+For more information about the files needed to identify the components in the images, see xref:../operating/examine-images-for-vulnerabilities.adoc#examine-images-for-vulnerabilities[Examining images for vulnerabilities].
+
+
+|`Failed to get the base OS information.`
+|Indicates that the target registry is unreachable on the network. The cause could be a firewall blocking `docker.io`, or an authentication issue preventing access.
```

Comment:
I think this description needs to be expanded. What does this mean from the vulnerability date perspective? Does it mean that CVE data for operating system-level packages is cannot be determined.
cc: @RTann

Reply:
This message means we were unable to determine the base operating system used for the image. This is not related to image registries at all